ses/index.cgi

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/perl
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+use 5.10.1;
+use strict;
+use warnings;
+
+use lib qw(.. ../lib ../local/lib/perl5);
+
+use Bugzilla ();
+use Bugzilla::Mailer qw( MessageToMTA );
+use Bugzilla::User ();
+use Bugzilla::Util qw( remote_ip );
+use JSON::XS qw( decode_json encode_json );
+use LWP::UserAgent ();
+use Try::Tiny qw( try catch );
+
```

Comment:
Since this is in sub directory, the error pages will never work properly. 
This isn't strictly needed, but it would be the "right" thing to do and probably good to do:

```perl
Bugzilla->error_mode(ERROR_MODE_DIE);
try {
    main();
} catch {
    warn "Error in SES handler: $_";
    respond(500 => 'Internal Server Error');
};
```